specification/loadtestservice/LoadTestService/client.tsp

```diff
@@ -32,6 +116,7 @@ interface AdministrationOperations {
   listTests is LoadTestAdministration.listTests;
   #suppress "@azure-tools/typespec-azure-core/byos" "It is limiting for users to have storage first before creating a load test also it will be breaking change for us."
   @clientName("BeginUploadTestFile", "python")
```

Comment:
Do pascal and snake casing matter for python client name?

Reply:
That is automatically handled by the generator (it converts this to `begin_upload_test_file`). Hence same casing is also not done for other client renames